src/poly/dcrt/matrix/dcrt_poly_matrix.rs

```diff
@@ -230,73 +232,6 @@ impl PolyMatrix for DCRTPolyMatrix {
         )
         .decompose()
     }
-
-    fn read_from_files<P: AsRef<Path> + Send + Sync>(
-        params: &<Self::P as Poly>::Params,
```

Comment:
@rkdud007 Can you keep these functions or just comment out them? While they are not used now, they will be definitely used soon to implement a function to read/write the obfuscation from/into files.